skabase/SKABaseDevice/SKABaseDevice.py

```diff
@@ -137,8 +148,8 @@ def get_device_attributes(self, with_value=False,
 
         # Cannot loop over the attr_list object (not python-wrapped): raises TypeError:
         # No to_python (by-value) converter found for C++ type: Tango::Attribute*
-        for index in range(len(attr_list)):
-
+        # for index in range(len(attr_list)):
+        for index in enumerate(attr_list):
```

Comment:
If you use `enumerate`, `index` will be a tuple, so line 153 is likely to break.